include/deal.II/base/time_stepping.h

```diff
@@ -78,8 +119,17 @@ namespace TimeStepping
    */
   enum embedded_runge_kutta_time_step
   {
+    /**
```

Comment:
Can you delete the descriptions of the 3 entries in the comment block above? No reason to exactly duplicate the information.

Reply:
I thought every Enumerator needs a table, sorry.

Reply:
Sorry, you misunderstood: please put the descriptions of the enum values back in, but remove the identical and now redundant text in the documentation 3 lines above the line I commented at. (Github doesn't want me to comment on a line you did not change)

Reply:
That means we will move the descriptions in parenthesis to the table, correct?

Reply:
Yes, exactly like you had it before.

Reply:
this is what I meant, thanks.